plot.py

```diff
@@ -3,9 +3,13 @@
 import matplotlib.pyplot as plt
 from matplotlib.colors import LogNorm
 import seaborn as sb
+plt.rcParams['font.size'] = 24
+plt.rcParams['font.family'] = 'Times New Roman'
 
 # Read in the file
-df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+# df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+df = pd.read_csv('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraWing/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
```

Comment:
Remove directory and filename out of command (for clarity)
e.g.
```dir = '/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraWing/'```
```filename = 'XYZ_S1.dat'
```pd.read_csv('dir+filename...```